### PR TITLE
Convert slug to lowercase and Create Telegram bot to send new event registration to the group 

### PR DESCRIPTION
**Pull request by michaelwp**
Convert slug to lowercase and Create Telegram bot to send new event registration to the group enhancement

### CODE REVIEW

**Comment by kei-001** [comment]
Convert slug to lowercase

**Comment by jokosu10** [comment]
pak @kei-001,
setelah saya coba untuk yang slug eventnya, masi belum sesuai. Gambar uji coba terlampir. 

cc mas @dikyarga 
Screen Shot 2019-04-13 at 09 59 20

**Comment by dikyarga** [comment]
coba saya cek yak @jokosu10 
https://github.com/id-mozilla/mozspace-jkt-backend/issues/5

File: common/telegram/tele.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const TelegramBot = require('node-telegram-bot-api');
+
+const token = '782076025:AAF_oyQlg-dbnBbHtUHDK1JPlG4m4sKVVN0'; // need token bot chat
```
Comment (mnirfan) [inline common/telegram/tele.js]: env nya di buat dari env variable ya (`process.env`)

**Review by mnirfan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**